draft-rescorla-tls-ctls.md

```diff
@@ -362,6 +442,10 @@ and so may have real problems.
 
 [[OPEN ISSUE: Do we want this at all?]]
 
+[[ XXX-RLB: My opening offer would be to not change this at all.  Just use the
+existing certificate types, but have the compression layer swap the cert_data in
+and out. ]]
+
```

Comment:
I don't think I quite understand how this works.

Reply:
Define a profile that says:

s/entire Certificate message/a single varint identifier that selects from a table/